TOOLS/odoo-tools.sh

```diff
@@ -1178,9 +1190,7 @@ if [ "$SCRIPT_MODE" = "updatetranslation" ]; then
     # Calling this script recursive with another option for maintenance mode
     sh -c "$0 maintenancemode ${TARGET_BRANCH} ${DBNAME} enable"
     echo "Create, temp database backup ${DBNAME} before update..."
-    ${INSTANCE_PATH}/TOOLS/db-tools.py -b ${BASEPORT69} -s "${SUPER_PASSWORD}" backup -d ${DBNAME} -f ${TMPFILENAME}_before.zip
-    # TODO: change this recursive call odoo_tools.sh later or use only pgdump for this temp backup
-    #sudo - postgres ${DBNAME} > ${TMPFILENAME}_before.sql
+    sh -c "$0 backup ${TARGET_BRANCH} ${DBNAME}"
     echo "Stopping odoo of instance ${DBNAME} ..."
     service ${DBNAME} stop
     # ------------------------------------------ ADMIN PREPARATIONS 1 - 2 - 3 END -------------------------------------
```

Comment:
Please switch to common format:
# ----- Admin Preparations 1 - 2 - 3 END
or (even better) avoid comments like this at all and do smaller comment like
# ----- Setting basic Variables
# ----- Creating needed Folders

and so on